packages/web-react/src/components/UNSTABLE_Slider/README.md

```diff
@@ -0,0 +1,69 @@
+# UNSTABLE Slider
+
+⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+Please use it with caution.
+
+## Basic usage
+
+```jsx
+import { useState } from 'react';
+import { UNSTABLE_Slider } from '@lmc-eu/spirit-web-react/components';
+
+const [value, setValue] = useState();
+
+const handleChange = (event) => {
+  setValue(Number(event.target.value));
+};
+
+<UNSTABLE_Slider id="slider" label="Slider" value={value} onChange={handleChange} />;
+```
+
+### API
+
+| Attribute        | Type                     | Default | Required | Description                                               |
+| ---------------- | ------------------------ | ------- | -------- | --------------------------------------------------------- |
+| `helperText`     | `string`                 | -       | ✕        | Custom helper text                                        |
+| `id`             | `string`                 | -       | ✓        | Input and label identification                            |
+| `isFluid`        | `bool`                   | `false` | ✕        | Whether the element spans to the full width of its parent |
```

Comment:
I am missing isLabelHidden (or how do we call it in other components)

isRequired, isDisabled

Reply:
`isLabelHidden` and `isRequired` have been added to the table. We don't have `isDisabled` yet, but it will be added in the future.